CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## [Unreleased]
 
+## [4.1.1] - 2022-11-10
```

Comment:
missing link here

Reply:
of course, forgot to update that part! Thanks :)